### ISSUE TITLE
Feature request: Command to delete manually added tracker from torrent

### ISSUE THREAD

**Issue by Speeddymon** [rakshasa/rtorrent#542]
I manually added a tracker today to one of my torrents and accidentally pasted the URL to the torrent file itself instead of the announce URL. I tried to overwrite this URL by adding the correct URL in its place, but ended up with both the correct URL and the incorrect URL in the tracker group.

Please add a command to delete trackers from the trackers view as well as a command to delete entire tracker groups.

**Comment by smaslennikov** [comment]
I agree, removing URLs is just as important as adding them. If there's a command for one, why not for the other as well?

Cheers!
